### PR TITLE
feat: add digital-twins generic context for new platform

### PR DESCRIPTION
**Pull request by junjou**

### CODE REVIEW

File: digital-twins/jsonld-contexts/observed_attributes.jsonld [hunk 1]
```diff
@@ -0,0 +1,9 @@
+  {
+    "@context": {
+        "name": "https://uri.fiware.org/ns/data-models#name",
+        "description": "https://uri.fiware.org/ns/data-models#description",
```
Comment (bobeal) [inline digital-twins/jsonld-contexts/observed_attributes.jsonld:4]: those 2 terms are already defined in the core context (with IRIs from well known ontologies, you can use them)

File: digital-twins/jsonld-contexts/observed_attributes.jsonld [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        "blueprint": "https://uri.fiware.org/ns/data-models#blueprint",
```
Comment (bobeal) [inline digital-twins/jsonld-contexts/observed_attributes.jsonld:5]: I think the `uri.fiware.org` namespace is no longer used. you can use the smart data models one instead.

File: digital-twins/jsonld-contexts/digital_twins_contexts.jsonld [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "@context": [
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_attributes.jsonld",
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_entity_types.jsonld",
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_relationships.jsonld",
```
Comment (bobeal) [inline digital-twins/jsonld-contexts/digital_twins_contexts.jsonld:5]: don't know what to think about the separation into different files (and btw, a relationship is an attribute)

File: digital-twins/jsonld-contexts/observed_attributes.jsonld (hunk 1 above)
Comment (junjou) [reply digital-twins/jsonld-contexts/observed_attributes.jsonld:4]: If i'm not mistaken, only description is defined in the core context 1.3 ("name" does not appear here => https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld)

Description is defined with: http://purl.org/dc/terms/description

Also I'm not able to find those "well known ontologies" from IRIs. Do you know a particular website or something? What about https://schema.org/name ?

File: digital-twins/jsonld-contexts/digital_twins_contexts.jsonld (hunk 3 above)
Comment (junjou) [reply digital-twins/jsonld-contexts/digital_twins_contexts.jsonld:5]: Ok maybe it isn't really necessary, I'll regroup them

File: digital-twins/jsonld-contexts/observed_attributes.jsonld (hunk 1 above)
Comment (bobeal) [reply digital-twins/jsonld-contexts/observed_attributes.jsonld:4]: Weird, I was pretty sure that `name` was defined in the core context and mapped to https://schema.org/name but no. And the ontology is schema.org.

Description comes from Dublin Core: https://www.dublincore.org/specifications/dublin-core/dcmi-terms/

File: digital-twins/jsonld-contexts/observed_attributes.jsonld [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "@context": {
+    "UseCase": "https://smartdatamodels/UseCase",
```
Comment (bobeal) [inline digital-twins/jsonld-contexts/observed_attributes.jsonld:3]: SDM namespace is more like this: https://github.com/smart-data-models/dataModel.WaterQuality/blob/master/context.jsonld

File: digital-twins/jsonld-contexts/observed_attributes.jsonld [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    "UseCaseConfig": "https://smartdatamodels/UseCaseConfig",
+    "name": "https://smartdatamodels.org/name",
```
Comment (bobeal) [inline digital-twins/jsonld-contexts/observed_attributes.jsonld:5]: better use https://schema.org/name (standard and we already use in some other contexts)

File: digital-twins/jsonld-contexts/digital_twins_compound.jsonld [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "@context": [
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_attributes.jsonld",
```
Comment (bobeal) [inline digital-twins/jsonld-contexts/digital_twins_compound.jsonld:3]: name of the context is no longer accurate. maybe call it `usecase.jsonld`, it is globally what it is about.

Comment (junjou) [reply digital-twins/jsonld-contexts/digital_twins_compound.jsonld:3]: agree

**Review by bobeal — APPROVED** [review]

**Pull request merged**